src/components/header/mobile-menu.ts

```
@@ -0,0 +1,105 @@
+/**
+ * Mobile Menu Manager Module
+ * Handles mobile menu opening/closing functionality
+ */
+
+export class MobileMenuManager {
+  private menu: HTMLElement | null = null;
+  private toggleButton: HTMLElement | null = null;
+  private overlay: HTMLElement | null = null;
+  private isOpen: boolean = false;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)